alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/binder/solr/sharding/SolrShardingMetrics.java

```diff
@@ -59,7 +59,8 @@ public void updateMetrics() {
                     setAndCreateMetricIfNotExists("lastIndexedTxId", shardState.getLastIndexedTxId(),
```

Comment:
Ignore this file. This is just formatting via Xenit style.

Reply:
Please do not make changes to it then. That pollutes the history of the file.